cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/IndexTest.scala

```diff
@@ -116,7 +116,7 @@ Create a composite index on the label `Person` and the properties `name` and `ag
 ###assertion=show
 //
 
-SHOW INDEXES
+SHOW INDEXES VERBOSE OUTPUT
```

Comment:
No

I had deliberately chosen to have without VERBOSE on one of them (indexes) and with on the other (constraints). Could have `BRIEF OUTPUT` if we want but I wanted to show both versions of the commands